docs/team/chester.adoc

```diff
@@ -0,0 +1,77 @@
+= Chester Sim - Project Portfolio
```

Comment:
File name should be chestersim.adoc (according to GitHub username)

Reply:
Thanks for the catch!